app/src/main/res/layout/clothing_item.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
General comments about this file:
1. I really can't understand from the file name (clothing_item) what's its purpose, maybe some more descriptive name?
2. there are a lot of strings, colors and dimensions which are hard-coded here instead of the designated xml files. Can you please put strings in strings.xml, colors in colors.xml, etc... ?